### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by shapirus** [goharbor/harbor-operator#1071]
Latest Harbor version is now 2.9.0, well past the 2.6.x series supported by the latest operator's 1.4.0-rc1 release candidate.

Is it expected to receive further updates?

**Comment by Sea-you** [comment]
I'm also concerned about the project status right now. We opted to use the operator instead of the vanilla charts to benefit from existing and future day2 possibilities. Would be nice to know what's the status here.

**Comment by wy65701436** [comment]
Yes, it is. Due to limited resources, we are doing our best to maintain this project. We welcome contributions from the community to help us drive releases, bug fixes, and version updates for Harbor.
